File: PRAKTIKUM MODUL 6/PRAK604_2310817220028_SheilaSabina.py
def eve(code, pesan1):
    for i in range(len(code)):
        if code[i] == ' ' and pesan1[i] == ' ':
            pesan1[i] = ' '
        elif code[i] == pesan1[i]:
            pesan1[i] = '*'
        else:
            pesan1[i] = '#'

def main():

    print("Input")
    code = input().rstrip('\n')
    pesan1 = input().rstrip('\n')

    print("\nOutput")
    if len(code) != len(pesan1):
        print("Panjang kalimat berbeda, pesan palsu")
        return

    pesan1_list = list(pesan1)
    eve(code, pesan1_list)
    pesan1 = ''.join(pesan1_list)

    bintang = pesan1.count('*')
    pagar = pesan1.count('#')

    print(pesan1)
    print(f"* = {bintang}")
    print(f"# = {pagar}")

    if bintang >= pagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

if __name__ == "__main__":
    main()

File: PRAKTIKUM MODUL 6/PRAK604_2310817220028_SheilaSabina.c
#include <stdio.h>
#include <string.h>

void eve(char code[], char pesan1[]) {
    for (int i = 0; i < strlen(code); i++) {
        if (code[i] == ' ' && pesan1[i] == ' ') {
            pesan1[i] = ' ';
        } else if (code[i] == pesan1[i]) {
            pesan1[i] = '*';
        } else {
            pesan1[i] = '#';
        }
    }
}

int main() {
    char code[200], pesan1[200];

    printf("Input\n");
    gets(code);
    code[strcspn(code, "\n")] = '\0';
    gets(pesan1);
    pesan1[strcspn(pesan1, "\n")] = '\0';

    printf("\nOutput");
    if(strlen(code) != strlen(pesan1)) {
        printf("\nPanjang kalimat berbeda, pesan palsu \n");
        return 0;
    }

    eve(code, pesan1);

    int bintang = 0, pagar = 0;

    for(int i = 0; i < strlen(code); i++) {
        if(pesan1[i] == '*') {
            bintang++;
        } else if(pesan1[i] == '#') {
            pagar++;
        }
    }

    printf("\n");
    printf("%s\n",pesan1);
    printf("* = %d\n", bintang);
    printf("# = %d\n", pagar);

    if (bintang >= pagar) {
        printf("Pesan Asli\n");
    } else {
        printf("Pesan Palsu\n");
    }

    return 0;
}